MyString.java

```diff
@@ -78,8 +124,26 @@ public static String randomStringOfLetters(int n) {
      * @return a string consisting of str1 minus all the characters of str2
      */
     public static String remove(String str1, String str2) {
-       //// Replace the following statement with your code
-        return null;
+        String str = "";
+        boolean add = true;
+        char[] arr = new char[str2.length()];
+        for(int i=0;i<str2.length();i++){
+            arr[i] = str2.charAt(i);
+        }
+        for (int i = 0; i < str1.length(); i++) {
+            for(int j = 0; j<arr.length; j++){
+                if(str1.charAt(i) == arr[j]){
+                    arr[j] = ' ';
+                    add = false;
+                    break;
+                }
+            }
+            if(add){
+                str+=str1.charAt(i);
+            }
+            add = true;
+        }
+        return str;

```

Comment:
Try and go over Strings, from reading the your submission I feel that this subject is not sitting well ..
If you need some clarifications send me an email